### PR TITLE
Rename some left-overs from 'license configuration' to 'license classifications'.

### PR DESCRIPTION
**Pull request by fviernau**
To consistently use the new terminology.

The changes can be  breaking, because
* command line options are renamed
* a parameter of the `rules.kts` is renamed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ A Kotlin script that enables the injection of how-to-fix texts in markdown forma
 | ------ | ----- | ---------------- | ------------- |
 | Kotlin script | Global | `$ORT_CONFIG_DIR/how-to-fix-text-provider.kts` | Empty (n/a) |
 
-#### [License configuration file](./docs/config-file-licenses-yml.md)
+#### [License classifications file](./docs/config-file-licenses-yml.md)
```
Comment (sschuberth) [inline README.md:251]: Should we then also rename `config-file-licenses-yml.md` to `config-file-license-classifications-yml.md` in yet another commit?

**Review by sschuberth — COMMENTED** [review]
Thanks for this!

File: README.md (hunk 1 above)
Comment (fviernau) [reply README.md]: Yes, makes sense!

**Review by sschuberth — APPROVED** [review]

**Comment by oheger-bosch** [comment]
I think the old name of the command line option is still referenced in the renamed config-file-license-classifications-yml.md file.

**Comment by fviernau** [comment]
> I think the old name of the command line option is still referenced in the renamed config-file-license-classifications-yml.md file.

Good catch!.. fixed it.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**